tests/cromwellapi/test-labels.py

```diff
@@ -22,3 +22,29 @@ def test_labels(cromwell_api, recording_mode):
         res = cromwell_api.labels(workflow_id=job["id"])
 
     assert isinstance(res, dict)
+    assert len(res["labels"]) > 1
+    assert sorted(list(res["labels"].keys())) == sorted(
+        ["Label", "cromwell-workflow-id", "secondaryLabel", "workflowType"]
```

Comment:
Maybe this is a stupid suggestion but should we define this list right before, then check that `len(res["labels"]) == len(the_list_of_label_keys)` instead of checking `len(res["labels"]) > 1`?

Reply:
yeah i think so